app/models/donation.rb

```diff
@@ -6,6 +6,7 @@ class Donation < ActiveRecord::Base
   belongs_to :event, inverse_of: :donations
 
   validates :donor, presence: true
+  validates_associated :donor, presence: true
```

Comment:
`#validates_associated` does not take options, it uses the validations defined in `Donor`, so it should be just:

```
validates_associated :donor
```

😀